source/includes/steps-create-index.yaml

```diff
@@ -11,7 +11,7 @@ level: 4
 ref: enter-index-name
 content: |
    In the dialog, enter the name of the index to create, or leave blank
-   to have MongODB create a default name for the index.
+   to have MongoDB create a default name for the index.
```

Comment:
"Default" isn't a technical term, and I believe it's common knowledge as a concept. Not worth a glossary entry or a dictionary definition, I think.